### PR TITLE
suthesh/mt5-icon 

### PR DESCRIPTION
**Pull request by suthesan-deriv**
# Description

added mt5 macos download icon in dmt5 page

Changes:

-   added mt5 macos download icon in dmt5 page

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/dmt5/_download-app.js [hunk 1]
```diff
@@ -20,6 +21,12 @@ const query = graphql`
         }
     }
 `
+
+const DownloadLinkWrapper = styled.div`
+    display: grid;
+    grid-template-columns : repeat(2, 1fr);
+    grid-gap: 0.8rem;
```
Comment (AbdallaMourad) [inline src/pages/dmt5/_download-app.js:28]: should we use `px` here

File: src/common/utility.js [hunk 2]
```diff
@@ -123,6 +123,7 @@ const mga_link_url =
     'https://authorisation.mga.org.mt/verification.aspx?lang=EN&company=a5fd1edc-d072-4c26-b0cd-ab3fa0f0cc40&details=1'
 
 const dmt5_windows_url = 'https://download.mql5.com/cdn/web/deriv.limited/mt5/deriv5setup.exe'
+const dmt5_macos_url = 'https://download.mql5.com/cdn/web/metaquotes.software.corp/mt5/MetaTrader5.dmg'
```
Comment (AbdallaMourad) [inline src/common/utility.js:126]: `dmt5_macos_url` should occur before `dmt5_windows_url`

File: src/common/utility.js [hunk 3]
```diff
@@ -138,6 +139,7 @@ export {
     binary_url,
     deriv_cookie_domain,
     dmt5_windows_url,
+    dmt5_macos_url,
```
Comment (AbdallaMourad) [inline src/common/utility.js:142]: Same here

File: src/pages/dmt5/_download-app.js (hunk 1 above)
Comment (suthesan-deriv) [reply src/pages/dmt5/_download-app.js:28]: Yup man, now coming to think of it I think px would be a better choice 😂

**Review by AbdallaMourad — APPROVED** [review]

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**